src/citrine/resources/measurement_run.py

```diff
@@ -105,3 +106,40 @@ class MeasurementRunCollection(ObjectRunCollection[MeasurementRun]):
     def get_type(cls) -> Type[MeasurementRun]:
         """Return the resource type in the collection."""
         return MeasurementRun
+
+    def list_by_spec(self, uid: Union[UUID, str], scope: str = 'id') -> Iterator[MeasurementRun]:
```

Comment:
also implement list_by_template?

Reply:
I was holding off on implementing it since it wasn't asked for explicitly. To me, adding `list_by_template` implies its implementation is efficient, but it's not. If the user would use the intermediate values (specs), then it would be better for them to write that code themselves so they can take advantage of those values.